admin_guide/image_policy.adoc

```diff
@@ -0,0 +1,154 @@
+[[admin-guide-image-policy]]
+= Image Policy
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[image-policy-overview]]
+== Overview
+
+You can control which images are allowed to run on your cluster using the ImagePolicy
+admission plugin (currently considered beta).  It allows you to control:
+
+ 1. the source of images: which registries can be used to pull images
+ 2. image resolution: force pods to run with immutable digests to ensure the image doesn't change due to a re-tag
+ 3. docker image label restrictions: force an image to have or not have particular labels
+ 4. image annotation restrictions: force an image in the integration docker registry to have or not have particular annotations
+
+
```

Comment:
Need an anchor and for the heading to be in title case:

[[image-policy-configuring-the-image-policy-admission-plug-in]]
Configuring the ImagePolicy Admission Plug-in